go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/mattn/go-sqlite3 v1.14.16
 	github.com/ory/dockertest/v3 v3.9.1
 	github.com/osmosis-labs/go-mutesting v0.0.0-20221208041716-b43bcd97b3b3
-	github.com/osmosis-labs/osmosis/osmomath v0.0.2
+	github.com/osmosis-labs/osmosis/osmomath v0.0.3-dev
```

Comment:
I tried calling it this:
https://github.com/osmosis-labs/osmosis/releases/tag/osmomath%2Fv0.0.2-20230103003823-2ac5d3561928

But `go.sum doesn't update for some reason:
https://sum.golang.org/lookup/github.com/osmosis-labs/osmosis/osmomath@v0.0.2-20230103003823-2ac5d3561928

Therefore, keeping this as my trial run to unblock CI